secboot/secboot_tpm_test.go

```diff
@@ -403,31 +385,18 @@ func (s *secbootSuite) TestUnlockVolumeUsingSealedKeyIfEncrypted(c *C) {
 		}, {
 			// tpm and encrypted device, successful activation, but
 			// recovery key non-nil is an unexpected state
-			tpmEnabled: true, hasEncdev: true, lockOk: true, activated: true,
+			tpmEnabled: true, hasEncdev: true, activated: true,
 			activateErr: &sb.ActivateWithTPMSealedKeyError{
 				RecoveryKeyUsageErr: fmt.Errorf("unexpected"),
 			},
 			expUnlockMethod: secboot.UnlockStatusUnknown,
 			err:             `internal error: volume activated with unexpected error: .* \(unexpected\)`,
 			disk:            mockDiskWithEncDev,
-		}, {
-			// activation works but lock fails, without encrypted device (lock requested)
-			tpmEnabled: true, lockRequest: true, activated: true,
-			err:  "cannot lock access to sealed keys: block failed",
-			disk: mockDiskWithUnencDev,
-		}, {
-			// happy case without encrypted device
-			tpmEnabled: true, lockOk: true,
-			disk: mockDiskWithUnencDev,
-		}, {
-			// activation works but lock fails, no encrypted device
-			tpmEnabled: true,
-			disk:       mockDiskWithUnencDev,
 		}, {
 			// tpm error, no encrypted device
 			tpmErr: errors.New("tpm error"),
-			err:    `cannot unlock encrypted device "name": tpm error`,
-			disk:   mockDiskWithUnencDev,
+			//err:    `cannot unlock encrypted device "name": tpm error`,
```

Comment:
should this be commented out ?

Reply:
Not sure the test case makes sense any more since attempt to poke TPM is done only after we find `name-enc`, but the test is using the non-encrypted device. We can remedy that later though.

Reply:
ah I totally forgot to fix this in my PR, but as mentioned I think it's not an issue